src/DateSingleInput/DateSingleInput.js

```diff
@@ -0,0 +1,91 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import { Box, Input } from '@quarkly/widgets';
+import { useOverrides } from '@quarkly/components';
+import { format, isValid } from 'date-fns';
+import DatePicker from '../DatePicker';
+import useInput from './hooks/useInput';
+import { overrides, propInfo, defaultProps } from './props';
+
+const DateSingleInput = ({
```

Comment:
We need to add support for required and name attributes for inputs and also add controls to propInfo for these props.